Mapper/BaseGroupedMapper.php

```diff
@@ -70,11 +70,17 @@ public function with($name, array $options = array())
             'collapsed' => false,
             'class' => false,
             'description' => false,
+            'label' => $this->admin->getLabelTranslatorStrategy()->getLabel($name, $this->getName(), 'group'),
             'translation_domain' => null,
             'name' => $name,
             'box_class' => 'box box-primary',
         );
 
+        // BC layer for manual label
+        if (strpos($name, '.')) {
+            $defaultOptions['label'] = $name;
+        }
```

Comment:
> Ho ok I got it !

Great! It's a bit ugly but you get injection and no BC-break, and it's a piece of code that will be removed so I think it's a good trade-off